src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.AddressBook;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.*;
+import java.lang.*;
+
+
+/**
+ * Lists all persons in the address book in sorted order to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
+            + "Displays all persons in the address book as a list sorted in alphabetical order with index numbers.\n\t"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.